includes/classes/admin/class-admin.php

```diff
@@ -168,6 +169,15 @@ public function add_pip_navbar() {
             }
         }
 
+        /**
+         * Adds the Pilo'Press ID to the body, to make the previews work everywhere in the Admin
+         */
+        public function add_pilopress_id_for_previews() {
+            ?>
+            <script>document.body.id = "pip";</script>
```

Comment:
Maybe we could try to add it with the [admin_body_class](https://developer.wordpress.org/reference/hooks/admin_body_class/) hook rather than with JS (

Reply:
It's a bit messy but it being an ID instead of a class ensures a higher [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) no matter what
I doubt it negatively impacts loading times much

Reply:
I did some test with a class and it seems to have the same result as with the ID. Even tho a selector with an ID would be indeed stronger, I think it's nicer to be the less intrusive as possible (you can only have an ID but multiple classes) and maybe some other plugins use this kind of method of changing the ID which could cause conflicts.

I think it's safer with the class but if there are cases where the selector isn't strong enough, we could try to find another way